### DISCUSSION TITLE
See people in voice channels before connecting

### DISCUSSION THREAD

**Discussion by Mati20187** [stoatchat/discussions#354] [Feature Suggestions]
Hi, it would be very helpful to be able to see people in voice channels before joining like this:
 
![image](https://user-images.githubusercontent.com/31066931/165945391-1dd43d0f-f459-4585-be30-719ce640f56d.png)

I don't think there's much to add here. That's how Discord, Mumble, TeamSpeak ... handle this and it would be great to see it in Revolt as well :)

**Comment by MisterE123** [comment]
yes, I was about to make this issue. please implement
